ci/pipelines/deploy-to-production.yml

```diff
@@ -194,6 +201,9 @@ groups:
       - deploy-ledger-to-prod
       - smoke-test-ledger-on-prod
       - ledger-pact-tag
+  - name: nginx-proxy
+    jobs:
+      - deploy-toolbox-to-prod
 
```

Comment:
I don't think we need this. We need it for deploy-to-staging and deploy-to-test because the push-to-staging/prod is conditional on the deploy-toolbox-to-staging/prod. We could always just check toolbox to see if the deploy to prod has been successful.

Reply:
I think it makes it clear that when a new version of nginx is pushed, this job is triggered, rather than having to know that its hiding in the toolbox job group

Reply:
Sure. To be consistent then we should do this for the nginx-forward-proxy and telegraf